src/utils/load-audio.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx createElementEntity */
+import { createElementEntity } from "../utils/jsx-entity";
+import { ProjectionMode } from "./projection-mode";
+import { VideoTexture } from "three";
+import { renderAsEntity } from "../utils/jsx-entity";
+import { loadAudioTexture } from "../utils/load-audio-texture";
+import { HubsWorld } from "../app";
+
+export function* loadAudio(world: HubsWorld, url: string) {
+  const { texture, ratio }: { texture: VideoTexture; ratio: number } = yield loadAudioTexture(url);
+
+  // TODO: VideoTexture.image must be content that be played
```

Comment:
video-system gets the video element off of obj.material.map.image but it doesn't have to, we can store the video element elsewhere so that we don't have to fake the texture like this.

Reply:
I think media files to play can be stored in component. I didn't have time to test in this cycle. I want to try in next cycle.